nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/main/java/org/apache/nifi/processors/kafka/GetKafka.java

```diff
@@ -218,11 +232,13 @@
     }
 
     public void createConsumers(final ProcessContext context) {
-        final String topic = context.getProperty(TOPIC).getValue();
+        topic = context.getProperty(TOPIC).getValue();
```

Comment:
I was just not brave enough to remove those codes. Confirmed it works without setting here. Thanks for pointing this out.